es_ES/_data/catalog/widgets.json

```diff
@@ -0,0 +1,1582 @@
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/FittedBox-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "AspectRatio",
+    "description": "Un widget que intenta dimensionar al hijo a una relación de aspecto específica. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/AspectRatio-class.html",
+    "image": "<svg viewBox='0 0 100 100'><defs><marker id='arrow-aspect-ratio' orient='auto-start-reverse' viewBox='0 0 1 1' markerWidth='3' markerHeight='3' refX='0.5' refY='0.5'><path d='M 1 0.5 L 0.5 0 L 0.5 1 z' fill='#ffffff'/></marker></defs><rect x='0' y='0' width='100' height='100' fill='#3949ab'/><rect x='25' y='35' width='50' height='30' fill='#ffffff'/><rect x='27.5' y='37.5' width='45' height='25' fill='#4dd0e1'/><line x1='31' y1='41' x2='69' y2='59' stroke='#ffffff' stroke-width='2' marker-start='url(#arrow-aspect-ratio)' marker-end='url(#arrow-aspect-ratio)'/></svg>"
+  },
+  {
+    "name": "ConstrainedBox",
+    "description": "Un widget que impone restricciones adicionales en su child. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/ConstrainedBox-class.html",
+    "image": "<svg viewBox='0 0 100 100'><defs><marker id='arrow-constrained-box' orient='auto-start-reverse' viewBox='0 0 1 1' markerWidth='3' markerHeight='3' refX='0.5' refY='0.5'><path d='M 1 0.5 L 0.5 0 L 0.5 1 z' fill='#ffffff'/></marker></defs><rect x='0' y='0' width='100' height='100' fill='#3949ab'/><rect x='20' y='30' width='50' height='40' fill='#ffffff'/><rect x='22.5' y='32.5' width='45' height='35' fill='#4dd0e1'/><path d='M63 23 v 40' stroke='#ffffff' stroke-width='1' stroke-linecap='round' stroke-dasharray='0.5 3.1' fill='transparent'/><path d='M77 23 v 54' stroke='#ffffff' stroke-width='1' stroke-linecap='round' stroke-dasharray='0.5 3.1' fill='transparent'/><path d='M13 63 h 50' stroke='#ffffff' stroke-width='1' stroke-linecap='round' stroke-dasharray='0.5 3.1' fill='transparent'/><path d='M13 77 h 64' stroke='#ffffff' stroke-width='1' stroke-linecap='round' stroke-dasharray='0.5 3.1' fill='transparent'/><path d='M50 25 L60 25' stroke='#ffffff' stroke-width='2' marker-end='url(#arrow-constrained-box)'/><path d='M90 25 L80 25' stroke='#ffffff' stroke-width='2' marker-end='url(#arrow-constrained-box)'/><path d='M15 50 L15 60' stroke='#ffffff' stroke-width='2' marker-end='url(#arrow-constrained-box)'/><path d='M15 90 L15 80' stroke='#ffffff' stroke-width='2' marker-end='url(#arrow-constrained-box)'/></svg>"
+  },
+  {
+    "name": "Baseline",
+    "description": "Un widget que posiciona a su hijo de acuerdo con la línea base del hijo. ",
```

Comment:
A widget that positions its child according to the child's baseline

Un widget que posiciona su hijo de acuerdo a la linea base de los hijos de este ¿?¿?

Reply:
Un widget que posiciona a su hijo de acuerdo a su linea base?